Civi/API/Subscriber/PermissionCheck.php

```diff
@@ -87,4 +87,26 @@ public function onApiAuthorize(\Civi\API\Event\AuthorizeEvent $event) {
     }
   }
 
+  /**
+   * check API for ACL permission.
+   * @param $apiRequest
```

Comment:
@eileenmcnaughton Thanks for your suggestion. I've updated the comment block as per the rules.